### PR TITLE
Adding Bulgarian language.

### PR DESCRIPTION
**Pull request by toni-kolev**

### CODE REVIEW

File: i18n.php [hunk 1]
```diff
@@ -214,4 +214,21 @@
         'undefined_count'         => '{1,21,31} %1% не определен|]1,Inf] %1% не определено',
         'skipped_count'           => '{1,21,31} %1% пропущен|]1,Inf] %1% пропущено',
     ),
+    'bg'    => array(
+        'snippet_context_choice'  => 'В сет <snippet_undefined><snippet_keyword>%1%</snippet_keyword> има недекларирани стъпки. Изберете в кой Context да бъдат създадени:</snippet_undefined>',
+        'snippet_proposal_title'  => '<snippet_undefined><snippet_keyword>%1%</snippet_keyword> има липсващи стъпки. Можете да ги създадете чрез:</snippet_undefined>',
+        'snippet_missing_title'   => '<snippet_undefined>Използвайте този снипет <snippet_keyword>--snippets-for</snippet_keyword> за да генерирате кода за следните стъпки <snippet_keyword>%1%</snippet_keyword> през конзолата:</snippet_undefined>',
```
Comment (martin-georgiev) [inline i18n.php]:
```suggestion
        'snippet_missing_title'   => '<snippet_undefined>Използвайте този снипет <snippet_keyword>--snippets-for</snippet_keyword>, за да генерирате кода за стъпки <snippet_keyword>%1%</snippet_keyword> през конзолата:</snippet_undefined>',
```

Comment (ciaranmcnulty) [reply i18n.php]: What do you think of this @toni-kolev?

Comment (pamil) [reply i18n.php:253]: @martin-georgiev please open a PR if that's a better translation, I'll merge it as is (not perfect Bulgarian support > no Bulgarian support).

**Comment by pamil** [comment]
Thanks, Toni! :1st_place_medal:

**Pull request merged**